### PR TITLE
Create .gitignore

### PR DESCRIPTION
**Pull request by will889889**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+.vscode/settings.json
+.vs/slnx.sqlite
```
Comment (john0312) [inline .gitignore:3]: Consider ignoring the entire .vs and .vscode directory.

If you've multiple ignores for a single application, consider adding a comment to group them together.

**Comment by john0312** [comment]
LGTM, Approved

**Comment by john0312** [comment]
Oh wait, please squash the 2 commits together.

**Comment by will889889** [comment]
Done

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#Ignore all project files of visual studio code.
+.vscode/
+
+#Ignore all project files of visual studio code.
+.vscode/
```
Comment (destinycan) [inline .gitignore:5]: looks like the duplicate part?

**Comment by will889889** [comment]
Done, thanks you.

**Pull request merged**